README.md

```diff
@@ -32,42 +10,40 @@ Description of the function/functions.
 - To run unit tests: `make test`
 - To lint the repo: `make lint`
 
-## Running Locally with Docker
+### Running the Lambda Function Locally with Docker
```

Comment:
This is about as close in the README as I can find to comment...

I think it could be helpful having a small section that explains the difference between running the flask app directly vs making `curl` requests to the lambda handler.  Said another way, documentation that can answer the question, "Why does curling to the lambda handler return JSON, but navigating to `http://localhost:5000` returns a 'normal' website response?"

In this section, it could be a good place to observe that 99% of development could be done against running the flask app directly.  It should be very rare that we have to think about the fact this is deployed via an AWS lambda + function URL.  Once those details are worked out, it should be nearly invisible that is happening, whereas normal devleopment would be thinking about Flask routes, website responses, etc.

Reply:
As per our discussion (@ghukill), it was decided that for development with Flask apps, running the app locally is the best way to view changes / feature updates made to the app. The instructions for invoking the Lambda function running in a Docker container might be confusing, so it was decided to remove these instructions (and plan to write a doc that describes deployment in more detail instead).